tests/BenchmarkDotNet.Tests/Mocks/MockFactory.cs

```diff
@@ -44,10 +51,45 @@ private static BenchmarkReport CreateReport(IConfig config)
             return new BenchmarkReport(benchmark, buildResult, buildResult, new List<ExecuteResult> { executeResult }, measurements, default(GcStats));
         }
 
+        [LongRunJob]
         public class MockBenchmarkClass
         {
             [Benchmark]
-            public void Foo() { }
+            public void Foo()
+            {
+            }
+
+            [Benchmark]
+            public void Bar()
+            {
+            }
+        }
+
+        public class MockHostEnvironmentInfo : HostEnvironmentInfo
+        {
+            public static MockHostEnvironmentInfo Default = new MockHostEnvironmentInfo
+            {
+                Architecture = "64bit",
+                BenchmarkDotNetVersion = "0.10.3.20170408-develop",
+                ChronometerFrequency = new Frequency(2531248),
+                Configuration = "RELEASE",
+                DotNetCliVersion = new Lazy<string>(() => "1.0.0"),
+                HardwareTimerKind = HardwareTimerKind.Tsc,
+                HasAttachedDebugger = false,
+                HasRyuJit = true,
+                IsConcurrentGC = false,
+                IsServerGC = false,
+                JitInfo = "RyuJIT-v4.6.1637.0",
+                JitModules = "clrjit-v4.6.1637.0",
+                OsVersion = new Lazy<string>(() => "Microsoft Windows NT 10.0.14393.0"),
+                ProcessorCount = 8,
+                ProcessorName = new Lazy<string>(() => "Intel(R) Core(TM) i7-6700HQ CPU 2.60GHz"),
+                RuntimeVersion = "Clr 4.0.30319.42000"
+            };
```

Comment:
Could we make these strings look like mock string. E.g. `0.10.3.20170408-develop` may confuse someone (it looks like an obsolete version). It's better to use something like `0.10.x-mock` here. Real versions of RyuJIT or Windows could accidentely coincide with real values and hide bugs, it's better use versions which are obvious fake (like `RyuJIT-v4.6.mock`).